quickstart/README.adoc

```diff
@@ -2,7 +2,7 @@
 
 == Starting the Server
```

Comment:
Shouldn't this page start with a Quickstart heading?
Then this header is level 2, but docker is level 4 - is this because you don't link the level 3 heading?